src/components/header/HeaderSearch.vue

```diff
@@ -0,0 +1,137 @@
+<script>
```

Comment:
note, this section of Search has not been modified at all. all changes are styling